controllers/seedgen_controller.go

```diff
@@ -521,7 +522,7 @@ func (r *SeedGeneratorReconciler) checkImagerStatus() error {
 		State ContainerState `json:"State"`
 	}
 
-	expectedStatus := "exited"
+	expectedStatuses := []string{"exited", "stopped"}
```

Comment:
@donpenney do you remember why we check exited only?
Why we can't create some file with status and read it?

Reply:
Because it's the standard way to check if a container you ran completed successfully.
In  any case, we can just ignore the status and check the exit code (as we already do in the next line).

Reply:
Seems like it doesn't really matter which of this status to change

Reply:
IIUC from what I've gathered so far, these 2 statuses from our point of view are the same. 

Specifically, this is what I managed to find out about `stopped` vs `exited`:

> stopped is general only a short state, that will be transitioned to exited very quickly, it is the state between we notice the container process died/exited and we cleanup all container resources (i.e. mounts, network, etc...). So in general I think exited is expected there.

I'm also in discussion with podman folks to see if we can track this different behavior between podman version 5.2.2 and 5.3.1 (since when using the same systemd version, the results are the same).

Reply:
Updated the PR's [description](https://github.com/openshift-kni/lifecycle-agent/pull/734#issue-2823737928) to reflect the root cause and the solution proposed.